app/app.go

```diff
@@ -569,34 +557,31 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	return app
 }
 
+// TODO(Tyler): is below needed?
+
 // MakeCodecs constructs the *std.Codec and *codec.LegacyAmino instances used by
 // Regenapp. It is useful for tests and clients who do not want to construct the
 // full Regenapp
-func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
-	config := MakeEncodingConfig()
-	return config.Codec, config.Amino
-}
+//func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
+//	cfg := MakeEncodingConfig()
+//	return cfg.Codec, cfg.Amino
+//}
 
 // Name returns the name of the App
 func (app *RegenApp) Name() string { return app.BaseApp.Name() }
 
 // BeginBlocker application updates every begin block
 func (app *RegenApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
 	resp := app.mm.BeginBlock(ctx, req)
-	events := app.smm.BeginBlock(ctx, req)
-	resp.Events = append(resp.Events, events...)
 	return resp
 }
 
 // EndBlocker application updates every end block
 func (app *RegenApp) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock {
 	resp := app.mm.EndBlock(ctx, req)
-	events, vals := app.smm.EndBlock(ctx, req)
-	if len(resp.ValidatorUpdates) > 0 && len(vals) > 0 {
+	if len(resp.ValidatorUpdates) > 0 { // TODO(tyler): is this still needed?
```

Comment:
I don't believe so. This was specifically added to prevent conflicts with our custom module manager iirc.